asv.conf.json

```diff
@@ -0,0 +1,22 @@
+
+{
+    "version": 1,
+    "project": "stardis",
+    "project_url": "https://tardis-sn.github.io/tardis",
+    "repo": ".",
+    "install_command": ["in-dir={env_dir} python -mpip install {wheel_file}"],
+    "uninstall_command": ["return-code=any python -mpip uninstall -y {project}"],
+    "build_command": [
+        "pip install git+https://github.com/tardis-sn/tardis.git@release-2023.06.18",
```

Comment:
can you make this a variable that is changeable?

Reply:
I can try

Reply:
Done, is that better now.